packages/fxa-auth-server/package.json

```diff
@@ -85,6 +89,7 @@
     "mozlog": "^3.0.2",
     "mysql": "^2.18.1",
     "mysql-patcher": "0.7.0",
+    "node-sass": "^6.0.1",
```

Comment:
Oops looks like node-sass is [deprecated](https://github.com/sass/node-sass#node-sass). Just a quick heads up that I'll be replacing it with [`sass`](https://www.npmjs.com/package/sass) package